README.md

```diff
@@ -3,14 +3,14 @@ NoisePy is a Python package designed for fast and easy computation of ambient no
 
 Disclaimer: this code should not be used "as-is" and not run like a blackbox. The user is expected to change local paths and parameters. Submit an issue to github with information such as the scripts+error messages to debug.
 
-Detailed documentation can be found at https://noisepy.readthedocs.io/en/latest/
+Detailed documentation can be found at https://noisepy.github.io/NoisePy/
 
-[![Documentation Status](https://readthedocs.org/projects/noisepy/badge/?version=latest)](https://noisepy.readthedocs.io/en/latest/?badge=latest)
+[![Documentation Status](https://github.com/noisepy/NoisePy/actions/workflows/notebook.yml/badge.svg)](https://noisepy.readthedocs.io/en/latest/?badge=latest)
```

Comment:
the badge link should be (notebooks, plural): 
https://github.com/noisepy/NoisePy/actions/workflows/notebooks.yml/badge.svg

Also the link should be to the new github pages jupyter book, not the the readthedocs page since that is getting retired: https://noisepy.github.io/NoisePy/

You can verify the `.md` is renedering properly and that links work using the "Display rich diff" button:

<img width="400" alt="image" src="https://github.com/noisepy/NoisePy/assets/10456546/ed7460f8-d822-4629-a288-16060d3026b6">